src/Controller/StripeWebhookEndpoint.php

```diff
@@ -244,19 +258,20 @@ private function onAccountUpdated(Event $event): string
      * @throws \Doctrine\ORM\ORMException
      * @throws \Doctrine\ORM\OptimisticLockException
      */
-    private function onPaymentIntentOrChargeCreated(Event $event): string
+    private function onChargeCreated(Event $event): string
     {
         $apiStripePayment = $event->data->object;
```

Comment:
For clarity, I suggest changing the variable name to `$charge` or `$stripeCharge`.